### PR TITLE
MrCoin exchange link

### PR DESCRIPTION
**Pull request by szerintedmi**
- added link to MrCoin from Buy/Sell page 
- refactored my orders display

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 04bd3e49ea677dc80860f81f8a7d6c53c2f8524a

https://app.netlify.com/sites/augmint/deploys/5afb98231f12b71101303998

File: src/modules/ethereum/gas.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const cost = {
     // w narrative: 46733 - 56693
 
     PLACE_ORDER_GAS: 200000,
-    MATCH_ORDERS_GAS: 100000,
+    MATCH_ORDERS_GAS: 150000,
```
Comment (reyrodrigez) [inline src/modules/ethereum/gas.js]: why did you increase the gas price?

File: src/containers/exchange/index.js [hunk 2]
```diff
@@ -70,15 +69,15 @@ class ExchangeHome extends React.Component {
...
+                                    header="Order book"
                                 />
                             </Pgrid.Column>
                         </Pgrid.Row>
                         <Pgrid.Row>
                             <TradeHistory
                                 trades={trades}
                                 userAccountAddress={userAccount.address}
-                                header="Trades history"
+                                header="My trade history"
```
Comment (reyrodrigez) [inline src/containers/exchange/index.js]: in general this should be `Your ...` everywhere. For consistency leave it `My trade history` for now and I'll create a ticket.

File: src/modules/ethereum/gas.js (hunk 1 above)
Comment (szerintedmi) [reply src/modules/ethereum/gas.js]: match ran out of gas on rinkeby a couple of times. Unsure why b/c after increasing this limit the tx  excuted under 100k. This is gas limit only btw, the default given to end users - the actual gas consumption will depend on the transaction. (and the gas price is a different param, its default comes from the actual block gas price)

**Pull request merged**